src/ExtendedView.js

```diff
@@ -1,28 +1,27 @@
 import './ExtendedView.css'
 
 function ExtendedView(props) {
-  const singleMovie = props.singleMovieData.find(movie => movie.id === parseInt(props.id))
-  console.log(singleMovie)
+
 
   return(
     <div className="extended-page">
       <button className="back" onClick={() => props.changeExtendedState()}>BACK</button>
         <div className="movie-poster">
           <div
             className="movie-image"
-            style={{backgroundImage: `url(${singleMovie.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}>
-            <p className="title">{singleMovie.title}</p>
-            <button className="rating">Rating: {((singleMovie.average_rating/10)*100).toFixed(0)}%</button>
+            style={{backgroundImage: `url(${props.singleMovie.backdrop_path})`, backgroundSize: "cover", height: "75vh"}}>
```

Comment:
I think for easier viewing, we should move each style property to it's own line, like how it would be done on a CSS file.

Reply:
Why is the styling in this file at all? Can you move it to a CSS file?